docs/EPS/temp.md

```diff
@@ -0,0 +1,12 @@
+# Arquitetura de Microserviços
```

Comment:
O que é esse arquivo "temp"?
E se for algo que deveria estar de fato aqui, você cita Circle CI na linha 6... não existe Circle CI no nosso projeto.